adapter_pipelines/submit.wdl

```diff
@@ -150,11 +150,15 @@ task stage_files {
   output {
     Array[File] http_requests = glob("request_*.txt")
     Array[File] http_responses = glob("response_*.txt")
+    # Make this task output a placeholder string here for confirm_submission so that they can be executed in order
+    String ready_to_be_confirmed = "true"
```

Comment:
I think it makes sense to use Cromwell's `Boolean` type here instead since the value is `true`